.gitignore

```diff
@@ -91,3 +91,5 @@ html-report-multichain/
 
 # UI Integration tests
 test/integration/config/assets
+
+.manifest_overrides.json
```

Comment:
I think it's `manifest-overrides.json` (with a dash rather than an underscore) - you can see there is a rule for that [here](https://github.com/MetaMask/metamask-extension/blob/3d58992cc8d24e058c4cead3c0caf21f8ff78561/.gitignore#L56)

Reply:
Oh! didn't even see that there. Yeah, the file name just needs to point to whatever is defined in `.metamaskrc`. Removed it!